docs/HACKING.md

```diff
@@ -497,6 +497,12 @@ Now create a fake Windows image:
 dd if=/dev/zero of=windows-dd bs=512 count=10000
 ```
 
+Move the image somewhere that the snap can access:
+
+```shell
+mv windows-dd /var/snap/maas/common/windows-dd
```

Comment:
This fails with permission denied if using multipass. We should either update the multipass command to `multipass exec dev-maas -- sudo bash` or add sudo to this one.

```suggestion
sudo mv windows-dd /var/snap/maas/common/windows-dd
```